functions/signals/ks_main.php

```diff
@@ -108,45 +112,44 @@ public static function generateImage($height)
 			}
 		}
 		
+		$radius_main_lamp = 3.5;
+		$radius_small_lamp = 2.25;
+		
 		$image = '
-			<g transform="translate(0 ' . $height . ')">
+			<g transform="translate(0 ' . $translation_height . ')">
 				<g>
-					<polygon style="&background;" points="6,1 34,1 34,59 6,59"/>
+					<polygon style="&background;" points="6,1 ' . $width-6 . ',1 ' . $width-6 . ',' . $height+1 . ' 6,' . $height+1 . '"/>
```

Comment:
You need parantheses around the `+` and `-` operations e.g. `' . ( $width - 6 ) . '`